CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+
+ADD_BIICODE_TARGETS()
+TARGET_COMPILE_OPTIONS(${BII_BLOCK_TARGET} INTERFACE -std=c++1y)
```

Comment:
I've installed your fork through git and bii and `c++1y` seems to work, but I'd like to use `c++14` to be more explicit. Since earlier versions of gcc and clang support this flag, it works as an assertion that a recent enough version is being used. On the other hand, it's ok if MinGW doesn't support `c++14`, but the tests pass. Let me know.